game/scripts/vscripts/Sections/Chapter2/SectionCreeps.ts

```diff
@@ -107,15 +107,14 @@ const onStart = (complete: () => void) => {
                     tg.textDialog(LocalizationKey.Script_2_Creeps_3, context => context[CustomNpcKeys.SlacksMudGolem], 8),
                     tg.immediate(_ => GridNav.DestroyTreesAroundPoint(godzSpawnLocation, 300, true)),
                     tg.wait(1),
-                    tg.spawnUnit(CustomNpcKeys.GodzMudGolem, godzSpawnLocation, DotaTeam.GOODGUYS, context => context[CustomNpcKeys.GodzMudGolem], true),
+                    tg.spawnUnit(CustomNpcKeys.GodzMudGolem, godzSpawnLocation, DotaTeam.GOODGUYS, CustomNpcKeys.GodzMudGolem, true),
                     tg.immediate(context => {
                         const godzMudGolem = context[CustomNpcKeys.GodzMudGolem]
                         setUnitPacifist(godzMudGolem, true);
                     }),
                     tg.setCameraTarget(context => context[CustomNpcKeys.GodzMudGolem]),
                     tg.textDialog(LocalizationKey.Script_2_Creeps_4, context => context[CustomNpcKeys.GodzMudGolem], 5),
                     tg.setCameraTarget(playerHero),
-                    tg.setCameraTarget(undefined),
```

Comment:
The camera is supposed to be unlocked. (we're still having discussions whether or not it should or not be locked, right now it sounds like we'll not be locking the camera all the time).

However, I would probably change that to pan the camera from Godz to the player quickly (I can do this, or if you want to, feel free)

Reply:
I've checked the later sections (CH4 specifically) and saw camera was still locked, so I figured it was meant to be locked here too.

I'll add the pan change